unit_tests/test_sct_events_continuous_events_registry.py

```diff
@@ -39,7 +39,7 @@ def populated_registry(self,
 
     def test_add_event(self,
                        registry: ContinuousEventsRegistry):
-        # continuous events are added on instantiation
+        # continuous events are added to the registry on event instantiation
```

Comment:
This line was added in your previous commit. Can you please update it there to reduce number of changes in this commit?

Reply:
Fixed (change moved to first commit)